variables.yml

```diff
@@ -12,7 +12,7 @@ networks:
     hex_chain_id: '0x507'
     chain_spec: alphanet
     block_explorer: https://moonbase-blockscout.testnet.moonbeam.network/
-    parachain_release_tag: v0.13.2 # must be in this exact format for links to work
+    parachain_release_tag: v0.14.2 # must be in this exact format for links to work
     parachain_sha256sum: 9e748a0c91d5ed5f999bae91ceed1343db0d9dda40270201abf00db067c86a71
```

Comment:
Thanks for update the version, but you need to also update the `sha256_sum` of the new image.

Reply:
Thank you @albertov19! Fixed